examples/kotlin/hello_world/BUILD

```diff
@@ -6,6 +6,7 @@ android_binary(
     name = "hello_envoy_kt",
     custom_package = "io.envoyproxy.envoymobile.helloenvoykotlin",
     manifest = "AndroidManifest.xml",
+    proguard_specs = ["//dist:proguard_rules"],
```

Comment:
Does this need to be updated? Can/should this be referencing something in the aar?

Reply:
Good point. Let me double check if this actually works if we don't reference it

I'll separate out the envoy mobile specifics

Reply:
It doesn't seem like bazel does the aar inspection + merging for us. We'll have to do this sub-optimal thing for now